### ISSUE TITLE
HTML and XML files.

### ISSUE THREAD

**Issue by ghost** [Brewtarget/brewtarget#646]
Hello :)
1. Export recipes to a file HTML doesn't work completely.
2.  Exported XML files are encoded in ISO-8859-1 by default, which results in incorrect display of recipe names for non-English users, which I am. Life would be simpler if xml files were encoded in utf-8 by default. No more question marks in recipe names.

Regards

**Comment by matty0ung** [comment]
Hi there,

If you could give us some more details on 1 (eg sample recipe where you are seeing problems), then we can gladly take a look. (I couldn't spot any obvious problems when I exported a trivial recipe, but maybe I didn't have enough ingredients in it.)

For 2, I'm afraid the character encoding is part of the BeerXML spec - in other words, switching to UTF-8 would make us less compliant with the standard (albeit that the BeerXML standard is flawed in many ways).  We had some longer discussion about this in https://github.com/Brewtarget/brewtarget/issues/624 but the short answer is that we are working on support for BeerJSON, which is a better file format that fully supports UTF-8.

**Comment by matty0ung** [comment]
Ah, yes, I see what you mean.

There is a new way to export to HTML (courtesy of @mattiasmaahl) which means that, instead of "Export to HTML" you go to "Print and preview" and, from there, you have the option to save as HTML.

So, we need to get rid of that right-click menu option, which, as you've spotted, doesn't do anything (most probably because it's trying to trigger the old code that got replaced) and, probably, add "Print and preview" as an option on the right-click menu instead.

I'll re-open this ticket so we've got something to remind us to make the fix.  Thanks for the spot!
